packages/trader/src/Modules/Trading/Containers/strike-param-modal.jsx

```diff
@@ -27,10 +27,8 @@ const StrikeParamModal = ({ is_open, toggleModal, strike, onChange, name, strike
                             i18n_default_text='If you buy a "<0>{{trade_type}}</0>" option, you receive a payout at expiry if the final price is {{payout_status}} the strike price. Otherwise, your “<0>{{trade_type}}</0>” option will expire worthless.'
                             components={[<strong key={0} />]}
                             values={{
-                                trade_type:
-                                    vanilla_trade_type === 'VANILLALONGCALL' ? localize('Call') : localize('Put'),
-                                payout_status:
-                                    vanilla_trade_type === 'VANILLALONGCALL' ? localize('above') : localize('below'),
+                                trade_type: contract_subtype,
```

Comment:
@kate-deriv please consider keeping the ternary expression since `contract_subtype` comes as non-localized value here

Reply:
But it should be non-localized, because we don't translate Call/ Put

Reply:
But I'll be safer to use `localize` in case if someone decide to translate, so I'll add back ternary